unit-tests/dds/librs.py

```diff
@@ -1,46 +1,23 @@
 # License: Apache 2.0. See LICENSE file in root directory.
-# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+# Copyright(c) 2022-4 Intel Corporation. All Rights Reserved.
 
-import pyrealdds
-from rspy import log, test
-from time import sleep
-import threading
-from rspy.timer import Timer
+from rspy import log as _log
```

Comment:
Why did you start using underscore? Was there an overlap with pyrealsense2 imports?

Reply:
librs imports all of pyrealsense2, which has a `log` function...